packages/react-native-renderer/src/ReactFiberConfigFabric.js

```diff
@@ -48,11 +48,27 @@ const {
   unstable_getCurrentEventPriority: fabricGetCurrentEventPriority,
 } = nativeFabricUIManager;
 
+import {getClosestInstanceFromNode} from './ReactFabricComponentTree';
+
+import {
+  getInspectorDataForViewTag,
+  getInspectorDataForViewAtPoint,
+  getInspectorDataForInstance,
+} from './ReactNativeFiberInspector';
+
 import {
   enableFabricCompleteRootInCommitPhase,
   passChildrenWhenCloningPersistedNodes,
 } from 'shared/ReactFeatureFlags';
 
+export {default as rendererVersion} from 'shared/ReactVersion'; // TODO: Consider exporting the react-native version.
+export const rendererPackageName = 'react-native-renderer';
+export const extraDevToolsConfig = {
+  getInspectorDataForInstance,
+  getInspectorDataForViewTag,
+  getInspectorDataForViewAtPoint,
+};
```

Comment:
This will require some changes on React Native side before the sync

Reply:
You are right, I thought we are calling `<renderer>.rendererConfig.getInspectorDataForViewAtPoint` on RN side, but we [actually use it from RDT global hook](https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Inspector/getInspectorDataForViewAtPoint.js#L70), so should be perfectly fine